conf/st2.conf.sample

```diff
@@ -278,7 +278,7 @@ version = 4
 # location of the logging.conf file
 logging = /etc/st2/logging.sensorcontainer.conf
 # Provider of sensor node partition config.
-partition_provider = {'name': 'default'}
+partition_provider = name:default
```

Comment:
Is this really the format for a valid dictionary variable?

Reply:
Yes. I've tripped on this several times. It's how oslo_config parses it. The k8s helm chart is the most recent place I tripped. The chart auto generates this setting based on how many sensor partitions a user requested. I've got tests in place there to make sure it uses this format because it does not work otherwise.

Reply:
It shows this format in the docs: https://docs.stackstorm.com/reference/sensor_partitioning.html
And here is where we generate that config in the helm chart: https://github.com/StackStorm/stackstorm-k8s/blob/master/templates/deployments.yaml#L1149-L1154

Reply:
OK, so it's a oslo_config format.  I'd be curious to know how white space is handled for multiple kv pairs and if this would become a issue for corner cases, but that's not what this PR is about.  Thanks for the explanation :+1: